serverless-heaven/serverless-webpack
# Adds the ability to specify an excludeFiles glob

## What did you implement:

Closes #433



## How did you implement it:

Fairly trivial, just used the ignore option in glob to exclude files matching the glob.

## How can we verify it:

1. Setup a serverless project with an index.test.js and an index.js that handle a route
2. Start it up and see that you get a warning about duplicate files matching for index
3. Now pull in this PR
4. add the following to your serverless.yml file in the custom > webpack section
`excludeFiles: **/*.test.js`
5. restart serverless
6. see that the warnings are now gone!

## Todos:


***Is this ready for review?:*** YES
***Is it a breaking change?:*** NO
